### DISCUSSION TITLE
Unexpected fuzz search results include those searched by separate characters

### DISCUSSION THREAD

**Discussion by zeroarst** [junegunn/fzf#3905] [Q&A]
Hi guys
I customize `_fzf_compgen_dir` function with below
```bash
_fzf_compgen_dir() {
  fd --type d --hidden --exclude ".git" . "$1"
}
```

and when I do
```bash
arst@ubuntu-4gb-hel1-1 ~
$ cd /

arst@ubuntu-4gb-hel1-1 /
$ cd nginx**<TAB> 
```

I found it includes paths searched by "n" "g" "i" "n" "x"... etc
![image](https://github.com/junegunn/fzf/assets/1834837/d9612f2d-d537-49fa-9884-2a5654a72984)

Which is not what I expect. I just want to see paths that include "nginx".
I checked the document and searched this discussion board. I could not find anything.
Maybe I don't know what is the right keyword to search for?

Does anyone know how to exlude those results?

**Comment by junegunn** [comment]
> I found it includes paths searched by "n" "g" "i" "n" "x"... etc

That's exactly what "fuzzy searching" does.

> I checked the document

Have you read https://github.com/junegunn/fzf#search-syntax? You can prefix your query with `'`.

**Reply by zeroarst** [reply]
Totally missed that! 
I added the exact option `--exact` (`-e`) and it works! 
```bash
export FZF_COMPLETION_OPTS='-e'
```
Thank you.
